src/main/java/com/emc/mongoose/storage/driver/kafka/KafkaStorageDriver.java

```diff
@@ -88,7 +112,41 @@ private void submitTopicOperation(PathOperation op, OpType opType) {
     }
   }
 
-  private void submitTopicCreateOperation() {}
+  private void submitTopicCreateOperation(final PathOperation op) {
+
+    final String topicName = op.item().name();
+    try {
+      val newTopic = new NewTopic(topicName, 1, (short) 1);
+      val createTopicsResult = adminClient.createTopics(Collections.singleton(newTopic));
+      val createTopicFuture = createTopicsResult.values().get(topicName).
+              whenComplete((result, exception) -> {
```

Comment:
Let's use some driver method reference instead of creating the handler instance every time